cms/static/cms/js/require-config.js

```diff
@@ -63,10 +63,13 @@
             "xblock": "coffee/src/xblock",
             "utility": "js/src/utility",
             "accessibility": "js/src/accessibility_tools",
-            "draggabilly": "js/vendor/draggabilly.pkgd",
             "URI": "js/vendor/URI.min",
             "ieshim": "js/src/ie_shim",
             "tooltip_manager": "js/src/tooltip_manager",
+            "modernizr": "js/vendor/afontgarde/modernizr.fontface-generatedcontent",
```

Comment:
This looks good for now. Very shortly I'll be removing these files from the repo, so then I'll update these paths to point to the installed Pattern Library versions instead, but we shouldn't hold up this PR for that.

BTW, do you know the difference between the ``modernizr`` file in the Pattern Library and this "generated content" version. Do you see a problem with replacing one with the other?